var/spack/repos/builtin/packages/libmatheval/guile-2.0.patch

```diff
@@ -0,0 +1,23 @@
+--- libmatheval-1.1.11/configure.in	2013-08-14 22:22:08.000000000 +0200
++++ libmatheval-1.1.11/configure.ac	2018-06-26 10:26:46.364866000 +0200
```

Comment:
This filename change is confusing. I usually like to diff two different directories, like `a/configure.in` and `b/configure.in`.